core/src/utils/gesture/swipe-back.ts

```diff
@@ -79,7 +79,7 @@ export const createSwipeBackGesture = (
   return createGesture({
     el,
     gestureName: 'goback-swipe',
-    gesturePriority: 40,
+    gesturePriority: 101,
```

Comment:
```suggestion
    /**
     * Swipe to go back should have priority over other horizontal swipe
     * gestures. These gestures have a priority of 100 which is why 101 was chosen here.
     */
    gesturePriority: 101,
```

We should probably update these values to use a const/enum, but this should be fine for now.